jetty-client/src/test/java/org/eclipse/jetty/client/HttpClientTLSTest.java

```diff
@@ -916,6 +963,7 @@ protected void service(String target, Request jettyRequest, HttpServletRequest r
 
         assertThrows(Exception.class, () -> client.newRequest("localhost", connector.getLocalPort()).scheme(HttpScheme.HTTPS.asString()).send());
 
+        byteBufferPool.asRetainableByteBufferPool().clear();
         await().atMost(5, TimeUnit.SECONDS).until(() -> leakedBuffers, is(empty()));
```

Comment:
I just noticed something unrelated to your change: the `leakedBuffers` variable should have a `CopyOnWriteArrayList`.